### ISSUE TITLE
superpmi.py can't run successfully with Python3

### ISSUE THREAD

**Issue by tannergooding** [dotnet/runtime#35574]
Attempting to run something like `python.exe -d .\scripts\superpmi.py collect "D:\tagoo\Repos\runtime\src\coreclr\tests\runtest.cmd" "x64 checked"`, currently results in:
```
SuperPMI collect
------------------------------------------------------------
Start time: 09:19:15
name 'coreclr_args' is not defined
Finished SuperPMI collect
Finish time: 09:19:15
Elapsed time: 0:00:00.003000
```

At the very least, this is because https://github.com/dotnet/runtime/blob/master/src/coreclr/scripts/superpmi.py#L536 is directly referencing `coreclr_args` when it should be using `self.coreclr_args`. However, `pylint` also reports all of the following which may also be issues waiting to happen:
![image](https://user-images.githubusercontent.com/10487869/80514341-36918580-8935-11ea-86a3-7438519fe3f1.png)

**Comment by tannergooding** [comment]
CC. @jashook, @BruceForstall; since you are the only two to have touched the file

**Comment by tannergooding** [comment]
Python 2.7 reached EOL on Jan 1st, 2020; so only Python 3+ is officially supported now. This was tested on Python 3.8.2

**Comment by jashook** [comment]
This seems like it is unrelated to python3 vs python2. This script actually only supported python 3.7+ as it uses asyncio

**Comment by jashook** [comment]
Seems like there have just been changes which broke the coreclr_args usage

**Comment by BruceForstall** [comment]
Fixed with #35822
